### ISSUE TITLE
npm install not working

### ISSUE THREAD

**Issue by wahid-dev1** [simonassank/react-native-bixolon-printer#1]
![image](https://user-images.githubusercontent.com/37960778/111080921-9e879500-8522-11eb-880c-1c4065c409ae.png)

**Comment by simonas-dev** [comment]
Yup, because it's not submitted to NPM registry. You'll have to include the source code into your project manually.

Note that there's no iOS support

**Comment by wahid-dev1** [comment]
Is there any other means to connect bixolon r200 react native  for both ios
and android

**Comment by simonas-dev** [comment]
None, I am aware of.

At least two years ago there were not. That's why I've made this library for just the Android native bindings. It was enough for our use case.

It's likely you'd have to write native bindings for iOS yourself or contract someone else.

**Comment by Pharaon3** [comment]
Hello.
I am new to React Native.
I want to use your library but don't know how to.
Can you lead me to test your code on my emulator?

Should I open a new project and integrate your library to that?
Thanks in advance.

**Comment by simonas-dev** [comment]
Hi,

I wasn't really working with RN either. Just wrote this native binding and integration was done by somebody else.

This might be useful https://reactnative.dev/docs/native-modules-android

Sorry, but you're on your own on this :D
